src/documents/TextToSpeechAPI.md

```diff
@@ -0,0 +1,580 @@
+* Chrome 33+ 
+* Chrome for Android 40+
+* Safari 7.1+
+
+## Creator ##
+
+Use the following function to create a Text To Speech component:
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>fluid.textToSpeech(options);</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                Instantiate the text to speech component. The methods of the instantiated component can be called to queue up and read texts.
+            </td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dl>
+                    <dt>options</dt>
+                    <dd>
+                        An optional data structure that configures the Text To Speech component, as described below.
+                    </dd>
+                </dl>
+            </td>
+        </tr>
+        <tr>
+            <th>Returns</th>
+            <td>The Text To Speech component</td>
+        </tr>
+        <tr>
+            <th>Examples</th>
+            <td>
+<pre>
+<code>
+var tts = fluid.textToSpeech({
+    utteranceOpts: {
+        volume: 1
+    }
+});
+</code>
+</pre>
+            </td>
+        </tr>
+        <tr>
+            <th>See also</th>
+            <td>
+                <a href="#utteranceopts-options">Utteranceopts Options</a>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Supported Methods ##
+
+### `queueSpeech` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textToSpeech.queueSpeech(text, interrupt, options);</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>queueSpeech</code> method allows you to append the text to the end of the queue to begin speaking after the other texts in the queue have been spoken. This method can also be used to remove all texts from the queue by passing in the `interrupt` argument with a true value.
+            </td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dl>
+                    <dt>text</dt>
+                    <dd>
+                        The string of the text to be appended to the end of the queue to be spoken.
+                    </dd>
+                    <dt>interrupt</dt>
+                    <dd>
+                        An optional boolean value. Passing in a true value to remove all texts from the queue. Passing in the false value doesn't interfere the component state.
+                    </dd>
+                    <dt>options</dt>
+                    <dd>
+                        An optional javascript object. To config the internal `SpeechSynthesisUtterance` instance. Refer to <a href="#utteranceopts-options">utteranceOpts Options</a> for the options that can be defined in this object.
```

Comment:
It's not clear what the relationship is between utteranceOpts and this method. Are you saying that anything that can go in utteranceOpts can also be used as an option to this method?